src/ANTLRInputStream.ts

```diff
@@ -132,9 +132,13 @@ export class ANTLRInputStream implements CharStream {
 	public getText(interval: Interval): string {
 		let start: number = interval.a;
 		let stop: number = interval.b;
-		if (stop >= this.n) stop = this.n - 1;
+		if (stop >= this.n) {
+			stop = this.n - 1;
+		}
```

Comment:
Personally I like a different rule, allowing non-brace single-line if statements.   In cases like this the braces add noise.

Reply:
tslint didn't enforce the multi-line statement. I manually fixed the layout since the line was already changing.

Reply:
But it forced the braces, right?   `curly: ignore-same-line` allows the original.